lib/manager/ansible-galaxy/__snapshots__/extract.spec.ts.snap

```diff
@@ -75,9 +75,21 @@ Array [
   },
   Object {
     "currentValue": "0.14",
-    "datasource": "galaxy",
-    "depName": "company.com:mygroup/ansible-base.git",
+    "datasource": "git-tags",
+    "depName": "mygroup/ansible-base",
```

Comment:
Should not be hit by the `galaxyRoleRegex` ( [namespace.role_name](https://docs.ansible.com/ansible/latest/galaxy/user_guide.html) ).